src/glb_main.c

```diff
@@ -21,6 +21,10 @@
 #include <sys/stat.h>  // for mkfifo()
 #include <fcntl.h>     // for open()
 #include <errno.h>
+#include <grp.h> // for getgrnam()
+
+/* Change this to the user under which to run */
+#define RUN_AS_USER "daemon"
```

Comment:
Here we have RUN_AS_USER independently defined in two different files. This does not look good.